app/Providers/AuthServiceProvider.php

```diff
@@ -2,6 +2,10 @@
 
 namespace App\Providers;
 
+use App\OAuth\WikiSocialiteServer;
+use App\OAuth\WikiSocialiteProvider;
+use Laravel\Socialite\One\TwitterProvider;
```

Comment:
Am I musing something, or are this and the following line unnecessary leftovers?

Reply:
Whoops. Fixed.